subsystem-drive/implementations/steer-modes.hpp

```diff
@@ -36,13 +36,24 @@ namespace sjsu::drive
 
         tri_wheel_router_arguments SpinSteering(drive_commands commands)
         {
-            // TODO: Calculate tri_wheel_router_arguments
-            return {};
+            tri_wheel_router_arguments temp;
+            temp.back.steer.speed  = 5; //setting the steer motors to the spin position
+            temp.left.steer.speed  = 5;
+            temp.right.steer.speed = 5;
+            temp.back.steer.angle = 120;// it's either 120 or 60 will have to test
+            temp.left.steer.angle = -120;
+            temp.right.steer.angle = -120;
+            // insert lerping for hub speed here
+            temp.back.hub.speed = commands.speed;
+            temp.left.hub.speed = -commands.speed;
+            temp.right.hub.speed = -commands.speed;
+
+            return temp;
```

Comment:
We wont be lerping in this class from my knowledge. I also think that these steer angles are incorrect, we have slight adjustments to the angles in the drive class, could you change these spin angles to match those? Then resolve your conflicts and merge.